core/src/main/java/com/novoda/accessibility/ActionMenuItem.java

```diff
@@ -0,0 +1,273 @@
+    @Override
+    public char getNumericShortcut() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setAlphabeticShortcut(char alphaChar) {
+        Log.w("!!!", "setAlphabeticShortcut(char) is not supported but is called by MenuInflater");
+        return this;
+    }
+
+    @Override
+    public char getAlphabeticShortcut() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setCheckable(boolean checkable) {
+        if (checkable) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    @Override
+    public boolean isCheckable() {
+        return false;
+    }
+
+    @Override
+    public ActionMenuItem setChecked(boolean checked) {
+        if (checked) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    @Override
+    public boolean isChecked() {
+        return false;
+    }
+
+    @Override
+    public ActionMenuItem setVisible(boolean visible) {
+        if (!visible) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    /**
+     * For our case, a menu item will always/only be visible if enabled.
+     */
+    @Override
+    public boolean isVisible() {
+        return isEnabled();
+    }
+
+    @Override
+    public ActionMenuItem setEnabled(boolean enabled) {
+        this.enabled = enabled;
+        return this;
+    }
+
+    @Override
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    @Override
+    public boolean hasSubMenu() {
+        return false;
+    }
+
+    @Nullable
+    @Override
+    public SubMenu getSubMenu() {
+        return null;
+    }
+
+    @Override
+    public ActionMenuItem setOnMenuItemClickListener(OnMenuItemClickListener menuItemClickListener) {
+        this.menuItemClickListener = menuItemClickListener;
+        return this;
+    }
+
+    @Nullable
+    @Override
+    public ContextMenu.ContextMenuInfo getMenuInfo() {
+        return null;
+    }
+
+    @Override
+    public void setShowAsAction(int actionEnum) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setShowAsActionFlags(int actionEnum) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setActionView(View view) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setActionView(int resId) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public View getActionView() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setActionProvider(ActionProvider actionProvider) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Nullable
+    @Override
+    public ActionProvider getActionProvider() {
+        return null;
```

Comment:
no `UnsupportedOperationException`?

Reply:
For the ones that were nullable, I returned null